tests/test_theory_parent.py

```diff
@@ -104,43 +108,6 @@ def test_instantiate(modeling_data):
     reduced_shear = mod.eval_reduced_tangential_shear(r_proj, z_cl, np.repeat(z_src, len(r_proj)))
     assert_allclose(reduced_shear, shear/(1.0-convergence), rtol=1.0e-12)
 
-    beta_s_mean = 0.9
-    beta_s_square_mean = 0.6
-    source_redshift_inf = 1000. 
-    shear_inf = mod.eval_tangential_shear(r_proj, z_cl, source_redshift_inf)
-    convergence_inf = mod.eval_convergence(r_proj, z_cl, source_redshift_inf)
-
-    #Tests with pre-fixed beta values
-    reduced_shear = mod.eval_reduced_tangential_shear(r_proj, z_cl, z_src, 'applegate14', beta_s_mean, beta_s_square_mean)
-    assert_allclose(reduced_shear, beta_s_mean * shear_inf/(1.0 - beta_s_square_mean / beta_s_mean * convergence_inf), rtol=1.0e-12)
-
-    reduced_shear = mod.eval_reduced_tangential_shear(r_proj, z_cl, np.repeat(z_src, len(r_proj)), 'applegate14', beta_s_mean, beta_s_square_mean)
-    assert_allclose(reduced_shear, beta_s_mean * shear_inf/(1.0 - beta_s_square_mean / beta_s_mean * convergence_inf), rtol=1.0e-12)
-
-    reduced_shear = mod.eval_reduced_tangential_shear(r_proj, z_cl, z_src, 'schrabback18', beta_s_mean, beta_s_square_mean)
-    assert_allclose(reduced_shear, (1. + (beta_s_square_mean / (beta_s_mean * beta_s_mean) - 1.) * beta_s_mean * convergence_inf) * (beta_s_mean * shear_inf / (1. - beta_s_mean * convergence_inf)), rtol=1.0e-12)
-
-    reduced_shear = mod.eval_reduced_tangential_shear(r_proj, z_cl, np.repeat(z_src, len(r_proj)), 'schrabback18', beta_s_mean, beta_s_square_mean)
-    assert_allclose(reduced_shear, (1. + (beta_s_square_mean / (beta_s_mean * beta_s_mean) - 1.) * beta_s_mean * convergence_inf) * (beta_s_mean * shear_inf / (1. - beta_s_mean * convergence_inf)), rtol=1.0e-12)
-
-    #Tests where the function computes the beta values
-    beta_s_mean = None
-    beta_s_square_mean = None
-
-    beta_s_square_test = compute_beta_s_square_mean(z_cl, 1000., mod.cosmo)
-    beta_s_test = compute_beta_s_mean(z_cl, 1000., mod.cosmo)   
-    reduced_shear = mod.eval_reduced_tangential_shear(r_proj, z_cl, z_src, 'applegate14', beta_s_mean, beta_s_square_mean)
-    assert_allclose(reduced_shear, beta_s_test* shear_inf/(1.0 - beta_s_square_test / beta_s_test * convergence_inf), rtol=1.0e-12)
-
-    reduced_shear = mod.eval_reduced_tangential_shear(r_proj, z_cl, np.repeat(z_src, len(r_proj)), 'applegate14', beta_s_mean, beta_s_square_mean)
-    assert_allclose(reduced_shear, beta_s_test * shear_inf/(1.0 - beta_s_square_test / beta_s_test * convergence_inf), rtol=1.0e-12)
-
-    reduced_shear = mod.eval_reduced_tangential_shear(r_proj, z_cl, z_src, 'schrabback18', beta_s_mean, beta_s_square_mean)
-    assert_allclose(reduced_shear, (1. + (beta_s_square_test / (beta_s_test * beta_s_test) - 1.) * beta_s_test * convergence_inf) * (beta_s_test * shear_inf / (1. - beta_s_test * convergence_inf)), rtol=1.0e-12)
-
-    reduced_shear = mod.eval_reduced_tangential_shear(r_proj, z_cl, np.repeat(z_src, len(r_proj)), 'schrabback18', beta_s_mean, beta_s_square_mean)
```

Comment:
Just to make sure I followed - these tests have been moved to `test_theory`, right?

Reply:
yes, just look for tests with `order1`/`order2` in it

Reply:
Thanks!